org.eclipse.winery.repository.ui/src/app/wineryRepository.component.ts

```diff
@@ -26,7 +26,7 @@ import { backendBaseURL } from './configuration';
 export class WineryRepositoryComponent implements OnInit {
     // region variables
     name = 'Winery Repository';
-    isBackendAvailable = false;
+    isBackendAvailable = -1;
```

Comment:
Can't we use `undefined` here?

Otherwise, it should be an emum. Otherwise, it is a [magic number](https://de.wikipedia.org/wiki/Magische_Zahl_(Informatik)), which shuld be avoided.